introduction.md

```diff
@@ -0,0 +1 @@
+# Please allow me to introduce myself https://www.youtube.com/watch?v=pkXIYgsvO0c
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.